packages/LUIS/bin/operations.json

```diff
@@ -259,13 +259,112 @@
         "description": "Returns the available endpoint deployment regions and URLs.",
         "name": "ListEndpoints"
     },
+    {
+        "method": "get",
+        "methodAlias": "list",
+        "target": [
+            "userazureaccounts"
+        ],
+        "command": "luis list userazureaccounts --armToken <string>",
```

Comment:
@AmirGeorge and @cleemullins are we expecting the user to have an armToken lying about?